tools/topology/topology2/platform/intel/dmic-generic.conf

```diff
@@ -102,6 +102,7 @@ IncludeByKey.PASSTHROUGH {
 					}
 				}
 				Object.Widget.gain.1 {
+					curve_duration 500000
```

Comment:
shall we set longer ramp for dmic capture? like 100ms? capture is something different with playback.

Reply:
The time need to conceal start transient in MTL and LNL for DMIC 48 kHz is about this length. It has been longer for cAVS platforms but since they use SOF v2.2 and tplg1 there is no need to address separately. So I'd keep this as 50 ms, same for all capture.